openhtf/output/callbacks/json_factory.py

```diff
@@ -39,7 +39,7 @@ def convert_to_dict(self, test_record):
     as_dict = data.convert_to_base_types(test_record)
     if self.inline_attachments:
       for phase, original_phase in zip(as_dict['phases'], test_record.phases):
-        for name, attachment in phase['attachments'].iteritems():
+        for name, attachment in phase['attachments'].items():
           original_data = original_phase.attachments[name].data
-          attachment['data'] = base64.standard_b64encode(original_data)
+          attachment['data'] = base64.standard_b64encode(original_data).decode('utf-8')
```

Comment:
Maintainers, please look at this, it was subtle. My understanding of the code was that the attachments were meant to be held as strings that held b64 encodings. Because of that, I chose to decode the byte string into a regular Python3 unicode string via utf-8. WDYT?

Reply:
SGTM, anything base64 encoded should make sense as utf-8 - your understanding is correct, that's the intention.